keras_core/backend/tensorflow/utils.py

```diff
@@ -0,0 +1,27 @@
+import tensorflow as tf
+
+
+def get_tensor_spec(t, dynamic_batch=False, name=None):
```

Comment:
We should only have one tf utils file. We already have `utils/tf_utils.py`. We need to consolidate in one of them. I think `backend/tf_utils.py` is probably the best choice: 1. explicit name (in general avoid `utils.py` it's too generic), 2. confined to the TF backend folder.

Reply:
Good idea, I've merged them into `tf_utils.py` under the TF backend folder.